source/release_notes/3.6.10-3.0.rst

```diff
@@ -0,0 +1,51 @@
+.. _3.6.10-3.0:
+
+================================================================================
+|PSMDB| |release|
+================================================================================
+
+:Date: October 31, 2018
+:Download: http://www.percona.com/downloads/percona-server-mongodb-3.6/
+:Installation: :ref:`install`
+
+Percona announces the release of Percona Server for MongoDB |release|
+on |date|. Download the latest version from the `Percona website
+<https://www.percona.com/downloads/percona-server-mongodb-3.6/>`_ or
+the `Percona Software Repositories
+<https://www.percona.com/doc/percona-server-for-mongodb/3.6/install/index.html>`_.
+
+|PSMDB| is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 3.6 Community Edition.  It
+supports MongoDB 3.6 protocols and drivers.
+
+|PSMDB| extends Community Edition functionality by including the :ref:`inmemory`
+storage engine, as well as several enterprise-grade features.  Also, it includes
+:ref:`mongorocks` storage engine, which is now deprecated.  |PSMDB| requires no
+changes to MongoDB applications or code.
+
+
```

Comment:
PSMDB-276 (Create tool to decrypt encrypted rollback files) is not mentioned. We have added a new tool to decrypt rollback files.
PSMDB-250 (nGram FTS implementation) is also not mentioned and I believe it's a community contribution so we should mention the name of the person who contributed this (check with @igorsol ).